eng/packages/http-client-csharp-mgmt/emitter/src/resource-detection.ts

```diff
@@ -279,7 +287,7 @@ function getSingletonResource(
   return singletonResource ?? "default";
 }
 
-function getResourceScope(model: InputModelType): ResourceScope {
+function getScopeOfResource(model: InputModelType): ResourceScope {
```

Comment:
nit: prefer getResourceScope since we have ResourceScope defined

Reply:
changed back.